chart/prometheus-msteams/values.yaml

```diff
@@ -54,6 +54,22 @@ customCardTemplate: ""
 #   {...}
 #   {{ end }}
 
+templateFile: "/etc/template/card.tmpl"
+
+extraConfigmapMounts: []
+# extraConfigmapMounts:
+#   - name: customConfigMap
+#     configMap: 
+#       name: customConfigMapName
+      
+## Additional Volume mounts
+extraVolumeMounts: []
+# extraVolumeMounts:
+#   - name: volume-name
+#     mountPath: /etc/custom_path
+
+
+
```

Comment:
dammit, you have to remove these blank lines, see the [linting CI job](https://github.com/prometheus-msteams/prometheus-msteams/pull/213/checks?check_run_id=2877920333). Afterwards I can go ahead and merge the PR.